docs/resources/managed_group.md

```diff
@@ -18,11 +18,11 @@ The managed group resource allows you to configure a Boundary group.
 ### Required
 
 - **auth_method_id** (String) The resource ID for the auth method.
+- **filter** (String) Boolean expression to filter the workers for this managed group.
```

Comment:
Probably shouldn't have been part of this PR but noticed `filter` incorrectly set as optional